recipes-devtools/clang/clang_git.bb

```diff
@@ -214,8 +214,9 @@ EXTRA_OECMAKE:append:class-target = "\
 "
 
 DEPENDS = "binutils zlib zstd libffi libxml2 libxml2-native ninja-native swig-native"
-DEPENDS:append:class-nativesdk = " clang-crosssdk-${SDK_SYS} virtual/nativesdk-cross-binutils nativesdk-python3"
-DEPENDS:append:class-target = " clang-cross-${TARGET_ARCH} python3 compiler-rt libcxx"
+DEPENDS:append:class-native = "llvm-native"
```

Comment:
this is not going to work in all case, because clang-native can also provide llvm-native, it will end up with dep loops then

Reply:
fixed in rhe new version